src/plugins/google-analytics/validator.js

```diff
@@ -0,0 +1,45 @@
+var _ = require('lodash');
+var assert = require('assert');
+var util = require('util');
+
+function validateDatasource(datasource) {
+  assert(datasource.keyFile != undefined,
+    util.format('Missing keyFile for Google Analytics datasource with id = %s. Add your key file to ./config/ and reference in the datasource with keyFile: "keyname"', datasource.id));
```

Comment:
Good idea. I think I will do this in a separate branch/fix and go through all string concatinations.